### ISSUE TITLE
Chapel support in highlight.js?

### ISSUE THREAD

**Issue by bradcray** [chapel-lang/chapel#16454]
It appears that Stack Overflow is changing from [Google Prettify to highlight.js](https://meta.stackexchange.com/questions/353983/goodbye-prettify-hello-highlight-js-swapping-out-our-syntax-highlighter?cb=1), yet it seems that [there is no Chapel support in highlight.js](https://github.com/highlightjs/highlight.js/blob/master/SUPPORTED_LANGUAGES.md).  This task is about adding Chapel support, either officially within highlight.js (ideally) or as an externally supported package (though... if we were to do this, would this mean that we wouldn't get highlighted on SO?)

**Comment by SakshiSarna** [comment]
I would like to work on this

**Comment by bradcray** [comment]
That would be great!  I don't think we know much about highlight.js yet on my team, but it looked pretty well documented from what I did see.  Let us know if/how we can help.

**Comment by bradcray** [comment]
It turns out that Discourse uses highlight.js as well, so this would be a double-win now that we have https://chapel.discourse.group/

**Comment by bradcray** [comment]
I'm taking a look at this as a study break this afternoon since the previous person indicating interest is now a ghost.

**Comment by bradcray** [comment]
I've added some initial Chapel support for highlight.js as a third-party package in https://github.com/highlightjs/highlight.js/pull/2806 and https://github.com/chapel-lang/highlightjs-chapel.  Unrelated to the PR and repo, I enabled Chapel highlighting for Discourse through the site's admin interface.  It looks as though enabling Chapel highlighting at StackOverflow will be a more significant effort as a third-party package, but that will be something to look into next.

The initial grammar I did is not particularly robust or complex, and could probably be improved, but I'm going to close this issue based on the fact that we now have some support.
